sql/migrate/migrate.go

```diff
@@ -263,37 +278,48 @@ func PlanFormat(fmt Formatter) PlannerOption {
 	}
 }
 
-// PlanWithoutChecksum disables the hash-sum functionality for the migration directory.
-func PlanWithoutChecksum() PlannerOption {
+// PlanWithChecksum allows setting if the hash-sum functionality
+// for the migration directory is enabled or not.
+func PlanWithChecksum(b bool) PlannerOption {
 	return func(p *Planner) {
-		p.sum = false
+		p.sum = b
```

Comment:
TBH, I don't like this. Because setting `PlanWithChecksum(true)` is a no-op.

Reply:
Kinda agree, but I thought about how to simplify the code by passing the flag directly, I can revert it. 

```
PlanWithChecksum(!MigrateFlag.DisableChecksum)
```

Reply:
I think this is a case where we "gain" 2 less lines of code without any real advantage. 

```go
if MigrateFlag.DisableChecksum {
  opts = append(opts, PlanWithoutChecksum())
}
```

Does not seem much simpler, except maybe getting rid of the `opts` variable.